src/Command/CopyCronTasksCommand.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Acquia\Cli\Command;
+
+use AcquiaCloudApi\Endpoints\Crons;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CopyCronTasksCommand.
+ */
+class CopyCronTasksCommand extends CommandBase {
+
+  protected static $defaultName = 'copy:cron';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Copy the cron tasks from one environment to other')
+      ->setAliases(['copy-cron'])
+      ->addArgument('source_app', InputArgument::REQUIRED, 'Alias of the source application in the format `app-name.env`')
+      ->addArgument('dest_app', InputArgument::REQUIRED, 'Alias of the destination application in the format `app-name.env`');
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int 0 if everything went fine, or an exit code
+   * @throws \Exception
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    // Get source app alias.
+    $source_alias = $input->getArgument('source_app');
+    $source_alias = $this->normalizeAlias($source_alias);
+    $source_alias = self::validateEnvironmentAlias($source_alias);
+
+    // Get destination app alias.
+    $destination_alias = $input->getArgument('dest_app');
+    $destination_alias = $this->normalizeAlias($destination_alias);
+    $destination_alias = self::validateEnvironmentAlias($destination_alias);
+
+    if ($source_alias === $destination_alias) {
+      $this->io->error('Source and Destination environments can not be same. Pls check.');
+      return 1;
+    }
+
+    // Get source and destination environments details based on alias.
+    $source_environment = $this->getEnvironmentFromAliasArg($source_alias);
+    $dest_environment = $this->getEnvironmentFromAliasArg($destination_alias);
+
+    // Get the cron resource.
+    $cron_resource = new Crons($this->cloudApiClientService->getClient());
+
+    $source_app_cron_list = $cron_resource->getAll($source_environment->uuid);
+    if ($source_app_cron_list->count() === 0) {
+      $this->io->error('There are no cron jobs in the source environment for copying.');
+      return 1;
+    }
+
+    // Ask for confirmation before starting the copy.
+    $answer = $this->io->confirm('Are you sure you\'d like to copy the cron jobs from ' . $source_alias . ' to ' . $destination_alias . '?');
+    if (!$answer) {
+      return 0;
+    }
+
+    $only_system_crons = TRUE;
```

Comment:
Is this going to change at some point?

Reply:
Not sure but there are some cron tasks which are default created when environment is provisioned example - DB back. 
And for those crons, api has `system` flag to `true` and thus not copying (skipping) those as those should already be available on the destination app.